### ISSUE TITLE
Change background color to the one more contrast one

### ISSUE THREAD

**Issue by havenousername** [DigatalArtDesign/static-environment-issues-report#56]
The ratio between background color and text should be > 7 for good contrast ratio. Take background color from here.

https://www.figma.com/file/Ct9RtsoL59ksR5HZE4hXjl/Untitled?node-id=30%3A2

**Comment by havenousername** [comment]
solved by #60
